common/src/main/java/com/wynntils/features/ItemScreenshotFeature.java

```diff
@@ -4,15 +4,18 @@
  */
 package com.wynntils.features;
 
-import com.google.common.collect.ImmutableList;
 import com.mojang.blaze3d.pipeline.MainTarget;
 import com.mojang.blaze3d.pipeline.RenderTarget;
 import com.mojang.blaze3d.platform.NativeImage;
 import com.mojang.blaze3d.systems.RenderSystem;
-import com.mojang.blaze3d.vertex.*;
+import com.mojang.blaze3d.vertex.BufferBuilder;
+import com.mojang.blaze3d.vertex.BufferUploader;
+import com.mojang.blaze3d.vertex.DefaultVertexFormat;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.blaze3d.vertex.Tesselator;
+import com.mojang.blaze3d.vertex.VertexFormat;
```

Comment:
Did you manually do this, or was this spotless? If it wasn't, maybe we should try to set up to avoid wild card imports?

Reply:
Spotless. Or possible ctrl-alt-O in IntelliJ. 

I usually collapse imports in the IDE, so I don't really care. But we should have a clear rule for wildcard imports. Either none at all (pros: it's a simple, clear, rule. You know there is a clear mapping between import lines and usage), or a limit (with more than X classes from package Y, always use wildcard).

I think the no-wildcards rule is attractive, but it is verbose and if you don't have import folding I can understand if it's too verbose.

Reply:
How was it not applied before this PR though, if it was done by spotless?

Reply:
Looks like spotless perhaps ignores imports. Then it is due to ctrl-alt-O (organize imports) in IntelliJ. I can configure it to do differently. What behavior do we want?

Reply:
Spotless should avoid wildcards imo.

Reply:
@kristofbolyai So you accept this change, but think it should be done all over the line? I'm perfectly fine with that. But let's do it in a separate PR.

Reply:
> @kristofbolyai So you accept this change, but think it should be done all over the line? I'm perfectly fine with that. But let's do it in a separate PR.

Yes, agreed.